### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][NVDA][MAS4.2.3][IntelliJ][Wipro]: Table commands are not working for tables present in Create Authentication files window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1488]
**Repro Steps:**
Open and Start IntelliJ IDEA.

1. Open your project with IntelliJ IDEA
2. Tab to Tools menu, point to "Azure", and then select "Azure Sign In".
3. In the Azure Sign In window, select "Automated" radio button, and navigate to "New" button and select it.
4. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
5. After sign In Create Authentication Files window will opened .
6. In "Create Authentication files window, press tab to navigate tables then try to access the table cells by using table commands(Ctrl+Alt+Arrow keys) then observe the NVDA Behavior

**Expected Result:**
**NVDA Behavior:**
By using below table command keys user should be able to access the table and NVDA should announced the Value present in the table cell then row position header name and column position
 Ctrl + Alt + Right Arrow-> Cell to right
Ctrl + Alt +Left Arrow->Cell to left
Ctrl + Alt +Down Arrow-> Cell below
Ctrl + Alt +Up Arrow->Cell above

**Actual Result:**
**NVDA Behavior:**
NVDA is not announcing anything by using table command keys and table keys are not working
**Jaws Behavior:**  Jaws is announcing as expected
Not Selected true
Column 1, Row 1
Not Selected true
Subscription name
Not Selected DevDiv Key Vault
Column 2
Subscription id
Not Selected bd62906c-0a81-43c3-a2f8-126e4cf66ada
Column 3

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![create authentication files-table](https://user-images.githubusercontent.com/22956130/36135078-b4397fa8-10ae-11e8-8a3e-27d28176d07f.jpg)
![create authentication files tables](https://user-images.githubusercontent.com/22956130/36135256-fd713282-10af-11e8-9dba-d5da5140ebf3.jpg)

**Comment by Eskibear** [comment]
duplicated with #1489

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749956
